src/app/tickets/my-tickets/my-tickets.component.ts

```diff
@@ -1,15 +1,79 @@
 import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Ticket } from 'src/app/_models/ticket';
+import { TicketService } from 'src/app/_services/ticket.service';
+import { ToastrService } from 'src/app/_services/toastr.service';
+import { Pagination, PaginatedResult } from 'src/app/_models/pagination';
 
 @Component({
   selector: 'app-my-tickets',
   templateUrl: './my-tickets.component.html',
   styleUrls: ['./my-tickets.component.css']
 })
 export class MyTicketsComponent implements OnInit {
+  mytickets: Ticket[];
+  pagination: Pagination;
+  ticketParams: any = {};
+  priorities = [
+    { value: '0', display: 'Low' },
```

Comment:
Co Cię tak ciągnie do angielskiego na widoku? :P Apkę mamy po polsku, musisz to pozmieniać :P